server/http/server.go

```diff
@@ -0,0 +1,74 @@
+package httpserver
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+
+	logging "github.com/ipfs/go-log/v2"
+)
+
+var log = logging.Logger("lassie/httpserver")
```

Comment:
I like this convention of prefixing with `lassie/`, maybe we should do this across the packages?

Reply:
I'm super down